projects/storefrontstyles/scss/themes/calydon/components/checkout/cart/_index.scss

```diff
@@ -0,0 +1 @@
+@import './cart-item';
```

Comment:
You need to add a trailing blank line on each one of the index files so prettier doesn't complain.